letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -557,12 +592,21 @@ if [ "$1" = "--le-auto-phase2" ]; then
           ln -s "$VENV_PATH" "$OLD_VENV_PATH"
         fi
         RerunWithArgs "$@"
+      # Otherwise bootstrap needs to be done manually by the user.
       else
-        error "Skipping upgrade because new OS dependencies may need to be installed."
-        error
-        error "To upgrade to a newer version, please run this script again manually so you can"
-        error "approve changes or with --non-interactive on the command line to automatically"
-        error "install any required packages."
+        # If it is because of the bootstrap, --non-interactive will be of no use.
```

Comment:
Same as https://github.com/certbot/certbot/pull/7268#discussion_r335716697 and https://github.com/certbot/certbot/pull/7268#discussion_r337693655.

```suggestion
        # If it is because bootstrapping is interactive, --non-interactive will be of no use.
```